snmp/datadog_checks/snmp/data/default_profiles/ibm-lenovo-server.yaml

```diff
@@ -0,0 +1,156 @@
+      - name: ibm.voltReading
+        OID: 1.3.6.1.4.1.2.3.51.3.1.2.2.1.3
+    metric_tags:
+      - column:
+          name: ibm.voltDescr
+          OID: 1.3.6.1.4.1.2.3.51.3.1.2.2.1.2
+        tag: ibm_volt_descr
+      - column:
+          OID: 1.3.6.1.4.1.2.3.51.3.1.2.2.1.11
+          name: ibm.voltHealthStatus
+        tag: ibm_volt_health_status
+  - MIB: IMM-MIB
+    table:
+      name: fanTable
+      OID: 1.3.6.1.4.1.2.3.51.3.1.3.2
+    symbols:
+      - name: ibm.fanSpeed
+        OID: 1.3.6.1.4.1.2.3.51.3.1.3.2.1.3
+        extract_value: '(\d+)%' # Example value : '74% of maximum'
+    metric_tags:
+      - column:
+          name: ibm.fanDescr
+          OID: 1.3.6.1.4.1.2.3.51.3.1.3.2.1.2
+        tag: ibm_fan_descr
+      - column:
+          OID: 1.3.6.1.4.1.2.3.51.3.1.3.2.1.10
+          name: ibm.fanHealthStatus
+        tag: ibm_fan_health_status
+  - MIB: IMM-MIB
+    table:
+      name: systemHealthSummaryTable
+      OID: 1.3.6.1.4.1.2.3.51.3.1.4.2
+    symbols:
+      - name: ibm.systemHealthSummary
+        constant_value_one: true
+    metric_tags:
+      - column:
+          name: ibm.systemHealthSummaryDescription
+          OID: 1.3.6.1.4.1.2.3.51.3.1.4.2.1.3
+        tag: ibm_system_health_summary_description
+      - column:
+          OID: 1.3.6.1.4.1.2.3.51.3.1.4.2.1.2
+          name: ibm.systemHealthSummarySeverity
+        tag: ibm_system_health_summary_severity
+  - MIB: IMM-MIB
+    table:
+      name: systemCPUVpdTable
+      OID: 1.3.6.1.4.1.2.3.51.3.1.5.20
+    symbols:
+      - name: ibm.systemCPUVpd
+        constant_value_one: true
+    metric_tags:
+      - column:
+          name: ibm.cpuVpdDescription
+          OID: 1.3.6.1.4.1.2.3.51.3.1.5.20.1.2
+        tag: ibm_cpu_vpd_description
+      - column:
+          OID: 1.3.6.1.4.1.2.3.51.3.1.5.20.1.11
+          name: ibm.cpuVpdHealthStatus
+        tag: ibm_cpu_vpd_health_status
+  - MIB: IMM-MIB
+    table:
+      name: systemMemoryVpdTable
+      OID: 1.3.6.1.4.1.2.3.51.3.1.5.21
+    symbols:
+      - name: ibm.systemMemoryVpd
+        constant_value_one: true
+    metric_tags:
+      - column:
+          name: ibm.memoryVpdDescription
+          OID: 1.3.6.1.4.1.2.3.51.3.1.5.21.1.2
+        tag: ibm_memory_vpd_description
+      - column:
+          OID: 1.3.6.1.4.1.2.3.51.3.1.5.21.1.8
+          name: ibm.memoryHealthStatus
+        tag: ibm_memory_health_status
+  - MIB: IMM-MIB
+    table:
+      name: powerTable
+      OID: 1.3.6.1.4.1.2.3.51.3.1.11.2
+    symbols:
+      - name: ibm.power
+        constant_value_one: true
+    metric_tags:
+      - column:
+          name: ibm.powerFruName
+          OID: 1.3.6.1.4.1.2.3.51.3.1.11.2.1.2
+        tag: ibm_power_fru_name
+      - column:
+          OID: 1.3.6.1.4.1.2.3.51.3.1.11.2.1.6
+          name: ibm.powerHealthStatus
+        tag: ibm_power_health_status
+  - MIB: IMM-MIB
+    table:
+      name: diskTable
+      OID: 1.3.6.1.4.1.2.3.51.3.1.12.2
+    symbols:
+      - name: ibm.disk
```

Comment:
`ibm.disk` seems very generic even with `ibm` prefix. It might conflicts with other `ibm`.

I think we should prefix everything `ibm.imm.` to:
- avoid conflicts
- can make it easier for user to search for metrics related to IMM (Integrated Management Module) related things.

Reply:
Makes sense, changed prefix to `ibm.imm`